swap_meet/vendor.py

```diff
@@ -1,2 +1,85 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+    
+    def get_by_id(self, item_id):
+        for item in self.inventory:
+            if item.id == item_id:
+                return item
+        return None
+        
+    def swap_items(self, other_vendor, my_item, their_item):
+
```

Comment:
Let's remove the blank line in order to keep the code block flush with the method definition line.
```suggestion
```